pages/common/git-tag.md

```diff
@@ -0,0 +1,19 @@
+# git tag
+
+> Create, list, delete or verify tags
+
+- List all tags
+
+`git tag`
+
+- Create a named tag pointing to the current commit
+
+`git tag {{tag_name}}`
+
+- Create a tag with the given message
```

Comment:
Would you say the message works more as a description of the tag, or a log entry (like a commit message) for the creation of the tag? Either way, the descriptions should probably specify it.